ocaml/xenopsd/lib/xenopsd.ml

```diff
@@ -418,8 +418,25 @@ let configure ?(specific_options = []) ?(specific_essential_paths = [])
     ~name:(Filename.basename Sys.argv.(0))
     ~version:Xapi_version.version ~doc ~options ~resources ()
 
+let log_raw_backtrace bt =
+  Option.iter
+    (fun slots ->
```

Comment:
Maybe too cryptic, but I believe you can omit `(fun slots ->` and `) slots` as the inner `Array.iteri` will have the correct type.